application/consumer-activemq-quarkus/src/main/java/webapp/tier/service/ActiveMqDeliverConsumer.java

```diff
@@ -0,0 +1,40 @@
+package webapp.tier.service;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.jms.JMSConsumer;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.TextMessage;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import webapp.tier.bean.MsgBean;
+import webapp.tier.util.MsgUtils;
+
+@ApplicationScoped
+public class ActiveMqDeliverConsumer extends ActiveMqConsumer {
+
+	@Inject
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6157602)